pkg/endpoint/endpoint_status_test.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// +build !privileged_tests
+// +build !privileged_tests,integration_tests
```

Comment:
I don't think that most of these `pkg/endpoint` tests are actually integration tests.

I believe that due to tech debt around the code structure, they introduce dependencies on the BPF subsystem, but that's more of a code / compile issue rather than being relevant to the actual test execution.

The right solution is to deal with that technical debt. That's been a desirable item for a long time though, and it's not at all reasonable to require this from you in this PR. A middle ground might be to use a different tag, rather than `integration_tests` we have something more apt. But that probably makes it a bit more of a hassle to manage running those tests via the make targets... so if that's still pretty painful then I'm willing to accept this.

Reply:
(I didn't look _too_ closely at these so I could also be off in my assessment here. If the comment seems off-base then feel free to ignore it).

Reply:
OK if I fix the endpoint tests in a follow-up PR?